XBVC/emitters/templates/nim_interface.jinja2

```diff
@@ -19,6 +19,14 @@ type
   s8* = int8
 
 type
+{% for enum in enumerations%}
+  {{enum.pascal_name}}* = enum
+  {% for val in enum.prefixed_camel_vals%}
+    {{val}} = {{loop.index - 1}},
+  {% endfor %}
+
```

Comment:
Is this blank line intentional?

Reply:
Yep!  Without this, there won't be a blank line between the enums.